app/models/concerns/audit_extensions.rb

```diff
@@ -103,8 +103,8 @@ def ensure_username
 
   def fix_auditable_type
     # STI Host class should use the stub module instead of Host::Base
-    self.auditable_type = "Host"          if auditable_type =~  /Host::/
-    self.associated_type = "Host"         if associated_type =~ /Host::/
+    self.auditable_type = "Host::Base"          if auditable_type =~  /Host::/
+    self.associated_type = "Host::Base"         if associated_type =~ /Host::/
```

Comment:
this is exactly the reason why we shouldn't align code with whitespaces :-) could you please remove them?